core/src/main/java/no/nav/dokdistdpi/consumer/rdist001/domain/Forsendelse.java

```diff
@@ -0,0 +1,18 @@
+package no.nav.dokdistdpi.consumer.rdist001.domain;
+
+import lombok.Value;
+
+import static java.lang.String.valueOf;
+
+@Value
+public class Forsendelse {
+
+	String forsendelseId;
+
+	public Forsendelse(Long forsendelseId) {
+		if (forsendelseId == null) {
+			throw new IllegalArgumentException("forsendelseId kan ikke være null");
+		}
+		this.forsendelseId = valueOf(forsendelseId);
```

Comment:
```suggestion
		this.forsendelseId = forsendelseId.toString();
```
`forsendelseId` kan ikke være `null` her